### DISCUSSION TITLE
Vise en chip med tall for antall valgte i combobox

### DISCUSSION THREAD

**Discussion by BBolneset** [digdir/designsystemet#2199] [Feature requests]
### Description

Combobox komponenten kan bli veldig lang vertikalt hvis det er valgt mange valg (multiple).
Forespørsel om feature er at når lengden på input-feltet fylles, kommer det en chip som viser antall resterende valgt.
For å se alle valgte må comboboxen åpnes. 
![Frame 7](https://github.com/user-attachments/assets/d7f661d6-c024-4bc9-bb2b-b6c5d255c372)

**Comment by mimarz** [comment]
Takker! 

Vi tar en gjennomgang å ser på det når resten av teamet er tilbake fra ferie :)

**Comment by Barsnes** [comment]
Hei, det vil bli eit par problem med denne løysingen:
- Det vil alltid bli layout shift når du åpner, kontra kun når nye valg blir tatt (som alltid vil skje)
- Personer som bruker skjermleser på page load vil ikkje få alle lest opp med ein gang

Det er også fordeler:
- Det er mindre visuelt som du kanskje ikkje trenger å sjå
- Tar mindre plass når den ikkje er i bruk

Det er mulig å bruke `hideChips` på `Combobox` dersom du vil vise chips på din eigen måte (dette ligger i next).

Tar gjerne i mot innspel på andre foredeler eller bakdeler, slik at me kan ta ein god avgjerelse på dette :)

**Comment by mrosvik** [comment]
Closing this discussion as this pattern introduces accessibility challenges that make it difficult to implement in an accessible way.
